src/workloads/sharding/MultiShardTransactionsWithManyNamespaces.yml

```diff
@@ -0,0 +1,237 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+
+GlobalDefaults:
+  Database0: &Database0 databaseForMultiShardTestingIndex0
+  Database1: &Database1 databaseForMultiShardTestingIndex1
+
+  ApproxDocumentSize: &ApproxDocumentSize 2000  # = 2kB
+  ApproxDocumentSize50Pct: &ApproxDocumentSize50Pct 1000  # = 1kB
+  DocumentCount: &DocumentCount 100_000
+
+  MaxPhase: &MaxPhase 4
+
+  RandomIdFilter: &RandomIdFilter {_id: {^RandomInt: {min: 1, max: *DocumentCount}}}
+
+  UpdateOperation: &UpdateOperation
+    OperationName: updateOne
+    OperationCommand:
+      Filter: *RandomIdFilter
+      Update: {$inc: {counter: 1}}
+
+  UpdateOperationDatabase0: &UpdateOperationDatabase0
+    <<: *UpdateOperation
+    Database: *Database0
+
+  UpdateOperationDatabase1: &UpdateOperationDatabase1
+    <<: *UpdateOperation
+    Database: *Database1
+
+Actors:
+- Name: CreateShardedCollection
```

Comment:
Did you mean to shard the collections also?

Reply:
No, I just need to make two different primary shards so the transactions are guaranteed to be multi-shard

Reply:
Then I think this should be renamed to `CreateDatabases` or something along that line since it doesn't create any (sharded) collections.

Reply:
Renamed the stage